ngraph/test/CMakeLists.txt

```diff
@@ -495,14 +495,46 @@ if (NGRAPH_ONNX_EDITOR_ENABLE)
         onnx/onnx_import_with_editor.in.cpp)
 endif()
 
-# SOURCE AND HEADERS FOR FRONTEND TESTING
+# SOURCE FOR FRONTEND TESTING
+
 file(GLOB FRONTEND_TESTS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/frontend/frontend_manager.cpp)
 set(SRC ${FRONTEND_TESTS_SRC} ${SRC})
 
 file(GLOB FRONTEND_SHARED_TESTS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/frontend/shared/src/*.cpp)
 file(GLOB FRONTEND_SHARED_TESTS_HDR ${CMAKE_CURRENT_SOURCE_DIR}/frontend/shared/include/*.hpp)
 set(SRC ${FRONTEND_SHARED_TESTS_SRC} ${SRC})
 
+# ---- PaddlePaddle FrontEnd testing ------
+if (NGRAPH_PDPD_FRONTEND_ENABLE)
+    find_package(Python3 COMPONENTS Interpreter)
```

Comment:
use `find_package(PythonInterp 3)`. 

Our CI and others pass `-DPYTHON_EXECUTABLE` which works only for `PythonInterp` while new cmkae module `Python3` does not take this value into account.

Reply:
Fixed